app/src/main/res/values-de/strings.xml

```diff
@@ -140,6 +140,8 @@
     <string name="permission_explain_text_contacts">Simlar liest die Nummern in deinem Telefonbuch, um andere Simlar-Nutzer zu finden.</string>
     <!-- Simlar ermöglichen, auf Fotos, Medien und Dateien zuzugreifen? -->
     <string name="permission_explain_text_storage">Simlar benutzt den Klingelton Deines Telefons und dieser liegt in den Medien und Dateien.</string>
```

Comment:
Not introduced by this commit but `Deines` should be `deines` to be consistent with the other texts.